cli/cli-interpreter/src/main/java/org/infinispan/cli/interpreter/statement/UpgradeStatement.java

```diff
@@ -27,7 +27,7 @@
 
    private enum Options {
       ALL, DUMPKEYS, SYNCHRONIZE, DISCONNECTSOURCE
```

Comment:
Did you want to remove `DUMPKEYS` here too?

Reply:
yes